CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ Version template:
  -->
  
 # Alfred Telemetry Changelog
+## [0.9.2] - 2022-12-23
+
+### Fixed
+* First call to Alfred Telemetry endpoint Solr always failing,Fix SolrSnapShot Metrics [#140]
```

Comment:
Fix what exactly? The next person reading this should now there was an incompatibility with the implementations we got, namely `AlfrescoReplicationHandler` removal and reference to that doc https://github.com/Alfresco/SearchServices/blob/master/search-services/alfresco-search/doc/architecture/solr-content-store-removal/00001-solr-content-store-removal.md